db/init.sql

```diff
@@ -4,13 +4,154 @@ CREATE DATABASE mm_db;
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
 CREATE TYPE user_role AS ENUM ('admin', 'teacher', 'student');
+CREATE TYPE attempt_state AS ENUM ('submitted', 'graded');
+-- NOTE(nrydanov): Need to think of other types together
+CREATE TYPE block_type AS ENUM ('task', 'markdown');
+
+CREATE TABLE unit_type (
+    id uuid PRIMARY KEY,
+    name varchar(64)
```

Comment:
I think in this case it is better to have a margin in case of long names (rare case, but also possible), so it is better to store `varchar(128)`

Reply:
Let's leave it how it is...